src/ReactiveUI/Mixins/DependencyResolverMixins.cs

```diff
@@ -19,6 +20,73 @@ namespace ReactiveUI
     /// </summary>
     public static class DependencyResolverMixins
     {
+        private static IReadOnlyList<RegistrationNamespace> defaultRegistrationNamespaces =
+            (RegistrationNamespace[])Enum.GetValues(typeof(RegistrationNamespace));
+
+        private static HashSet<RegistrationNamespace> registrationNamespacesToInitialize =
+            new HashSet<RegistrationNamespace>(defaultRegistrationNamespaces);
+
+        /// <summary>
+        /// Platforms or other registration namespaces for the dependency resolver to consider when initializing.
+        /// </summary>
+        public enum RegistrationNamespace
```

Comment:
Would prefer this to be a separate non-nested enum.

Reply:
I considered that and didn't know in which file it would live - is a neighboring file okay or should it live in some other folder?